assets/Technology-Tag-Icons/icons.tsx

```diff
@@ -0,0 +1,120 @@
+export const OffshoreWindIcon = () => (
```

Comment:
for all technology tag icons, add the following props `props: { fill: string | undefined; stroke: string | undefined; }`. This will allow us to change the color of the SVG (so that Ethan can change them to navy!). Then for all fill and stroke (except for when fill="none"), use the props e.g. fill={props.fill}. I will comment where you should do that for the first 2 icons! Below is a screenshot to help.

<img width="642" alt="Screenshot 2024-11-11 at 11 00 11 AM" src="https://github.com/user-attachments/assets/bd197ac7-f71a-4b20-a871-41072e0459d3">